unstructured/ingest/README.md

```diff
@@ -1,4 +1,5 @@
-# Batch Processing Documents
+# Batch Processing Documents [DEPRECATED]
+For the latest approach, go to: [v2](./v2)
```

Comment:
I guess not quite deprecated until we add all of v2, right?

Reply:
warning of "soon to be deprecated"?

Reply:
I guess the README should be considered fully deprecated because I don't want users coming here for docs on how to add a connector using this once the PR is merged in.